Plugins/BackbonePluginTemplatePlugin/BackbonePluginTemplatePluginGenerator.cs

```diff
@@ -76,6 +89,44 @@ private Task AddProjectsToSolution()
                 new[] { projectInfo, projectTestInfo });
         }
 
+        private Task AddInternalNugetServerToNugetConfig()
+        {
+            var nugetConfigFileName = "NuGet.config";
+            var nugetConfigFilePath = Path.Combine(SolutionDirectory, nugetConfigFileName);
+
+            var nugetConfigModification = CreateInternalNugetServerModification();
+            return _nugetConfigurationManipulator.ModifyNugetConfiguration(nugetConfigFilePath, nugetConfigModification);
+        }
+
+        private NugetConfigurationModification CreateInternalNugetServerModification()
+            => new NugetConfigurationModificationBuilder(_nugetPackageSourceManipulator)
+                .AddPackageSource(CreateInternalNugetServer())
+                .Build();
+
+        private NugetPackageSource CreateInternalNugetServer()
+            => new NugetPackageSource("Internal Nuget Server", "https://nuget.messerli.ch/v3/index.json");
```

Comment:
This can be a `private static readonly` field

Reply:
Addressed in: [Change methods to private static readonly fields](https://github.com/messerli-informatik-ag/meta-generator/pull/22/commits/2c63ff2574fd87f5e82c56698f614f7757671d5c)